streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredSessionStore.java

```diff
@@ -112,7 +110,7 @@ public void close() {
         Objects.requireNonNull(key, "key cannot be null");
         final Bytes bytesKey = keyBytes(key);
         return new MeteredWindowedKeyValueIterator<>(
-            inner.findSessions(
+            wrappedStore().findSessions(
```

Comment:
This is ugly -- we should have access to the wrapped store directly via inheritance. (also below)